src/DynamoCore/Models/HomeWorkspaceModel.cs

```diff
@@ -391,7 +439,20 @@ private void OnUpdateGraphCompleted(AsyncTask task)
 
             OnEvaluationCompleted(e);
 
+            if (EngineController.IsDisposed) return;
+
             EngineController.ReconcileTraceDataAndNotify();
+
+            // Refresh values of nodes that took part in update.
+            foreach (var modifiedNode in updateTask.ModifiedNodes)
+            {
+                modifiedNode.RequestValueUpdateAsync(scheduler, EngineController);
+            }
+
+            scheduler.Tasks.AllComplete(_ =>
+            {
+                OnRefreshCompleted(e);
+            });
```

Comment:
Hmm... This is pretty open-ended.  If you identify a solution, I'd be happy to pursue it...